Makefile

```diff
@@ -32,6 +32,9 @@ run_dev_no_genesis: build_docker
 run_dev: build_docker
 	docker-compose -f docker-compose/dev.yml -f docker-compose/genesis.yml -f docker-compose/run.yml up
 
+run_docs: build_docker
```

Comment:
In general it would be a good thing to generate static HTML files, place them to some folder so that we can put them onto our website.